spec/models/manageiq/providers/kubernetes/container_manager/refresh_parser_spec.rb

```diff
@@ -744,5 +744,222 @@
           :status_reason           => nil
         })
     end
+
+    it "tests volume without claim" do
+      expect(parser.send(
+        :parse_persistent_volume,
+        RecursiveOpenStruct.new(
+          :metadata => {
+            :name              => 'test-volume',
+            :uid               => '66213621-80a1-11e5-b907-28d2447dcefe',
+            :resourceVersion   => '448015',
+            :creationTimestamp => '2015-12-06T11:10:21Z'
+          },
+          :spec     => {
+            :capacity    => {
+              :storage => '10Gi'
+            },
+            :hostPath    => {
+              :path => '/tmp/data01'
+            },
+            :accessModes => ['ReadWriteOnce'],
+          },
+          :status   => {
+            :phase => 'Available'
+          }
+        )
+      )).to eq(
+        {
+          :name                    => 'test-volume',
+          :ems_ref                 => '66213621-80a1-11e5-b907-28d2447dcefe',
+          :ems_created_on          => '2015-12-06T11:10:21Z',
+          :namespace               => nil,
+          :resource_version        => '448015',
+          :type                    => 'PersistentVolume',
+          :status_phase            => 'Available',
+          :access_modes            => 'ReadWriteOnce',
+          :capacity                => 'storage=10Gi',
```

Comment:
@alongoldboim correct, but currently the parser of Persistent Volume does not return a hash so I can't change this test. I think this PR is ready, so it will be better to make this change in #7955 according to the change in the parser that is made there (once this PR will be merged). 
cc @simon3z